pymongo/client_session.py

```diff
@@ -900,12 +900,15 @@ def _start_retryable_write(self):
         self._check_ended()
         self._server_session.inc_transaction_id()
 
-    def _update_read_concern(self, cmd):
+    def _update_read_concern(self, cmd, sock_info):
         if (self.options.causal_consistency
                 and self.operation_time is not None):
             cmd.setdefault('readConcern', {})[
                 'afterClusterTime'] = self.operation_time
         if self.options.snapshot:
+            if sock_info.max_wire_version < 13:
+                raise ConfigurationError(
```

Comment:
This could be InvalidOperation or ConfigurationError. We seem to use both in this file for various "this thing is not supported" errors. Do you have a preference?

Reply:
`ConfigurationError` makes more sense to me.